system/ee/legacy/libraries/typography/Markdown/Michelf/MarkdownExtra.php

```diff
@@ -160,9 +160,9 @@ protected function doExtraAttributes($tag_name, $attr, $defaultIdValue = null, $
 		$attributes = array();
 		$id = false;
 		foreach ($elements as $element) {
-			if ($element{0} == '.') {
+			if ($element[0] == '.') {
```

Comment:
These need to be converted back to `{}` as well.

Reply:
Fixed on latest commit.